rc/filetype/makefile.kak

```diff
@@ -33,7 +33,7 @@ add-highlighter shared/makefile/content default-region group
 add-highlighter shared/makefile/comment region '#' '$' fill comment
 add-highlighter shared/makefile/evaluate-commands region -recurse '\(' '\$\(' '\)' fill value
 
-add-highlighter shared/makefile/content/ regex ^[\w.%-]+\h*:\s 0:variable
+add-highlighter shared/makefile/content/ regex ^[\w.%-/]+\h*:\s 0:variable
```

Comment:
What about windows style paths, e.g. `abc\\def`? I don't think we need to whitelist allowed characters here, we can simplify the regex with a simple-stupid:

```suggestion
add-highlighter shared/makefile/content/ regex ^\S+\h*:\s 0:variable
```

And if we want to be more accurate and not highlight trailing whitespace:

```suggestion
add-highlighter shared/makefile/content/ regex ^(\S+\h*:)\s 1:variable
```

Reply:
I initially didn't look into the syntax specification for make, but your suggestion made me research a bit more. I looked at the [OpenBSD make](https://man.openbsd.org/make) and [GNU make](https://www.gnu.org/software/make/manual/make.html) manuals.

Apparently there can be [multiple whitespace separated targets](https://www.gnu.org/software/make/manual/make.html#Multiple-Targets) and different target separators (OpenBSD's make has `:`, `::` and `!`). I'll think of a solution and update the pull request.

Thanks for making me think :-)